resources.yaml

```diff
@@ -280,37 +257,54 @@ deployment:
     image: htcondor-secondary
     secondary_htcondor_cluster: true
   training-hts-:
-    count: 3
+    count: 2
     flavor: c1.c28m225d50
     start: 2024-03-04
     end: 2024-03-08
     group: training-hts-fr
     image: htcondor-secondary
     secondary_htcondor_cluster: true
-  training-gie-:
+  training-path-gen-march-24:
```

Comment:
I'm not sure why this ID was since ever truncated and if there was a reason for this. So please watch out for errors.